### PR TITLE
Bump plugin-grunt-tasks, remove CSS

### PR DESCRIPTION
**Pull request by jdevalk**
## Context

* This pull relies on [this pull](https://github.com/Yoast/wordpress-seo/pull/15550/files) being merged into Yoast SEO.

## Summary

This PR can be summarized in the following changelog entry:

* Plugin no longer loads its own CSS file.

## Relevant technical choices:

* Added one CSS class to the new admin styling classes, dropped the CSS file from this plugin.

## Test instructions

This PR can be tested by following these steps:

* Update, see the Yoast meta box -> News SEO  content looks good.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,8 @@ node_modules/*
 /.idea
 /artifact
 artifact.zip
+assets/*.min.js
+assets/*.min.js.map
 phpunit.xml
 css/dist
```
Comment (IreneStr) [inline .gitignore:10]: Can be removed

File: Gruntfile.js [hunk 2]
```diff
@@ -73,9 +64,6 @@ module.exports = function( grunt ) {
 			},
 			grunt: "Gruntfile.js",
 		},
-		sassFiles: {
-			"css/dist/admin-metabox-<%= pluginVersionSlug %><%= pluginAssetSuffix %>.css": [ "css/src/admin-metabox.scss" ],
```
Comment (IreneStr) [inline Gruntfile.js:77]: I think 'pluginAssetSuffix' on line 19 can be removed as well. Or is it used somewhere else as well?

File: classes/wpseo-news.php [hunk 3]
```diff
@@ -209,17 +207,6 @@ public function flatten_version( $version ) {
...
 	}
 
-	/**
-	 * Enqueues the plugin scripts.
-	 */
-	public function enqueue_scripts() {
-		global $pagenow;
-
-		if ( $pagenow === 'post.php' || $pagenow === 'post-new.php' ) {
-			wp_enqueue_style( 'wpseo-news-admin-metabox-css', plugins_url( 'css/dist/admin-metabox-' . $this->flatten_version( WPSEO_NEWS_VERSION ) . WPSEO_CSSJS_SUFFIX . '.css', WPSEO_NEWS_FILE ), [], WPSEO_NEWS_VERSION );
```
Comment (IreneStr) [inline classes/wpseo-news.php:219]: Is flatten_version (line 202) used anywhere else, or can it be removed as well?

File: Gruntfile.js (hunk 2 above)
Comment (jdevalk) [reply Gruntfile.js:77]: True!

File: classes/wpseo-news.php (hunk 3 above)
Comment (jdevalk) [reply classes/wpseo-news.php:219]: Yes!

**Pull request merged**